.dart_tool/package_config_subset

```diff
@@ -54,6 +54,10 @@ path
 2.12
 file:///Users/thenamesoak/.pub-cache/hosted/pub.dev/path-1.8.2/
```

Comment:
These package locations only work on your local machine. We need to figure out how to solve this.